src/services/GooglePlacesService.ts

```diff
@@ -113,6 +120,31 @@ class GooglePlacesService {
       });
   }
 
+  // refreshes the invalid placeID, may also result in NOT_FOUND error
+  refreshPlaceID(placeID: string): Promise<string> {
+    return axios
+      .get('https://maps.googleapis.com/maps/api/place/details/json?', {
+        params: {
+          place_id: placeID,
+          fields: 'place_id',
+          key: process.env.REACT_APP_GOOGLE_PLACES_API_KEY,
+        },
```

Comment:
How come the `refresh URL` is the same as URL in `getRestaurantDetails`?

If `getRestaurantDetails` resulted in a NOT_FOUND won't this one be guaranteed to be a NOT_FOUND as well, since the URL is the exact same?

Reply:
Just kidding, maybe we can write an explanation for other developers summarizing this point below?
https://developers.google.com/places/web-service/place-id#save-id

Reply:
Which point did you want me to add in particular?

Reply:
Oh like for me it wasn't trivial to see that even though the URL is the same, when you only specify the `place_id` it does the refresh. But if you add more fields, then it won't do the refresh/

![image](https://user-images.githubusercontent.com/49849754/108160634-4afa6680-709e-11eb-962b-8f22febfdc01.png)
^Keyword **only**